frontend/viewer/src/lib/history/HistoryView.svelte

```diff
@@ -28,63 +32,67 @@
     }
   }
 </script>
-<Toggle let:on={open} let:toggleOn let:toggleOff on:toggleOn={load}>
-  <Button on:click={toggleOn} icon={mdiHistory} variant="fill-light" color="info" size="sm">
-    <div class="sm-form:hidden" class:hidden={small}>
-      View History
-    </div>
-  </Button>
-  <Dialog {open} on:close={toggleOff} {loading} persistent={loading} class="w-[700px]">
-    <Button on:click={toggleOff} icon={mdiCloseCircle} class="absolute right-2 top-2 z-40" rounded="full"></Button>
-    <div slot="title">History</div>
-    <div class="m-6 grid gap-x-6 h-[50vh]" style="grid-template-columns: auto 4fr;">
 
-      <div class="flex flex-col gap-4 overflow-y-auto">
-        <div class="border rounded-md">
-          {#if !history || history.length === 0}
-            <div class="p-4 text-center opacity-75">No history found</div>
+<Dialog bind:open on:open={load} {loading} persistent={loading}>
+  <Button on:click={() => open = false} icon={mdiClose} class="absolute right-2 top-2 z-40" rounded="full"></Button>
+  <div slot="title">History</div>
+  <div class="m-4 mt-0 grid gap-x-6 gap-y-1 overflow-hidden" style="grid-template-columns: 250px 1fr; grid-template-rows: auto minmax(0,100%)">
```

Comment:
I'm worried about if there's 250px on mobile

Reply:
Yes, you're right. It behave much better by simply not defining columns at all 😆.
Ultimately we'd probably like to go back and forth between the list and the contents, but...not today.